docs/get-started/create-app.md

```diff
@@ -131,7 +154,7 @@ tensor([[[0.9594, 0.9373],
 
 ## Deploy to JCloud
 
-JCloud offers free CPU and GPU instances to host Jina project. Let's deploy your first project to JCloud:
+JCloud offers free CPU and GPU instances to host Jina project. Let's deploy your first project to JCloud (Note that JCloud is only available for flow projects):
```

Comment:
```suggestion
\`\`\`admonition
:class: important

At present, JCloud is only available for Flows. We are currently working on supporting Deployments.
\`\`\`

JCloud offers free CPU and GPU instances to host your Jina project. Let's deploy your first project:
```

GH won't let me put my _actual_ suggestion in (it won't render), so I've had to put backslashes in front of the backticks. My actual suggestion doesn't include the backticks (`\`)